spec/support/matchers/wait_for_ajax.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+puts "Loading spec/support/matchers/wait_for_ajax.rb"
+
+module WaitForAjax
+
+  def wait_for_ajax
+    Timeout.timeout(Capybara.default_max_wait_time) do
+      loop until finished_all_ajax_requests?
+    end
+  end
+
+  def finished_all_ajax_requests?
+    page.evaluate_script("jQuery.active").zero?
+  end
+
+end
+
+RSpec.configure do |config|
+  config.include WaitForAjax, type: :js
```

Comment:
Oh I see the idea now - it might be sufficient to move this file up a level (from /matchers into /support).  Right now the spec runs are finding the `wait_for_ajax` method is undefined.